src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -319,5 +370,30 @@ public void Start()
             new Address("0x9FC3da866e7DF3a1c57adE1a97c9f00a70f010c8"), // Extra data as: Titan (titanbuilder.xyz)
             new Address("0x0b92619DdE55C0cbf828d32993a7fB004E00c84B"), // Extra data as: Builder+ www.btcs.com/builder
         };
+
+        private struct BlockData
```

Comment:
As it will always end up on heap due to multithreading, maybe better to make it a class and use ObjectPool?

Probably still miniscule impact compared to those strings though.

Reply:
Changed to pooling; is big enough to bother me 😉

Reply:
Pool the strings!